lobby-db/src/main/resources/db/migration/V1.10.00.20190330__lobby_user_table.sql

```diff
@@ -0,0 +1,54 @@
+create table lobby_user
+(
+    id              serial primary key,
+    username        character varying(40) not null unique,
+    password        character varying(60),
+    email           character varying(40) not null,
+    date_created    timestamptz           not null default current_timestamp,
+    last_login      timestamptz,
+    admin           boolean               not null default false,
+    bcrypt_password character(60) check (char_length(bcrypt_password) = 60)
+);
+
+-- alter table user owner to triplea_lobby;
+alter table lobby_user
+    owner to postgres;
+alter table lobby_user
+    add constraint lobby_user_pass_check check (password IS NOT NULL OR bcrypt_password IS NOT NULL);
+
+
+comment on table lobby_user is 'The table storing all the information about Lobby TripleA users.';
+comment on column lobby_user.id is 'Synthetic PK column';
+comment on column lobby_user.username is 'Defines the in-game username of everyone.';
+comment on column lobby_user.password is 'The legacy MD5Crypt hash of the password. The length of the hash must always be 34 chars. Either password or bcrypt_password must be not null.';
+comment on column lobby_user.email is 'Email storage of every user. Large size to match the maximum email length. More information here: https://stackoverflow.com/a/574698.';
+comment on column lobby_user.date_created is 'The timestamp of the creation of the account.';
+comment on column lobby_user.last_login is 'The timestamp of the last successful login.';
+comment on column lobby_user.admin is 'The role of the user, controls privileges. If true the user is able to ban and mute other people.';
+comment on column lobby_user.bcrypt_password is 'The BCrypt-Hashed password of the user, should be the same as the md5 password but in another form. The length of the hash must always be 60 chars. Either password or bcrypt_password must be not null.';
+
+
+insert into lobby_user (username, password, email, date_created, last_login, admin)
+select username, password, email, joined, lastLogin, admin
+from ta_users;
```

Comment:
Shouldn't the "old" table get deleted after this?

Reply:
I initially forgot, good catch! I was then waffling a bit yesterday after wondering about rollback. I'd agree best to drop it so the rename is complete: 2139de19f